src/drivers/gitlab.js

```diff
@@ -37,7 +37,7 @@ class Gitlab {
   }
 
   async repoBase() {
-    if (this._detected_base) return this._detected_base;
+    if (this.detectedBase) return this.detectedBase;
```

Comment:
could still keep the underscore prefix though if the intention was to be a private var.

Reply:
More conventional than anything else

Reply:
The most idiomatic alternative would be using a [private field](https://en.wikipedia.org/wiki/Memoization), but that's just a proposal.

Reply:
We use that variable to [memoi[]e](https://en.wikipedia.org/wiki/Memoization) the function result, so it can be considered private.

Reply:
it does not have to be private either but should be exposed in other drivers for consisitency, being very purist we should do a closure of request here with the detectedbase... not needed IMHO

Reply:
Using a closure sounds tempting, but would add a considerable maintenance overhead and nobody cares about our internal API, yet. 😅  Let's leave it be for now.

Reply:
@casperdcl, what do you think?

Reply:
yes not really massive priority to do this at [enterprise level](https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition) yet, fine as-is